strr-base-web/app/components/connect/form/field/Index.vue

```diff
@@ -18,7 +18,9 @@ defineProps({
 })
 
 const normalize = () => {
-  model.value = normalizeInput(model.value)
+  if (typeof model.value === 'string') {
+    model.value = normalizeInput(model.value)
```

Comment:
Just wondering if using the .trim accomplishes the same thing? The only thing it doesn't do is replace undefined with an empty string

https://vuejs.org/guide/essentials/forms.html#trim

Reply:
oooh nice yeah I'll try that